packages/main/src/ComboBox.ts

```diff
@@ -61,277 +68,34 @@ import ComboBoxItem from "./ComboBoxItem.js";
 import Icon from "./Icon.js";
 import Popover from "./Popover.js";
 import ResponsivePopover from "./ResponsivePopover.js";
-import List from "./List.js";
+import List, { ClickEventDetail } from "./List.js";
```

Comment:
The recommendation is to have two lines:

import List from "./List.js"; // imported by value
import type { ClickEventDetail } from "./List.js"; // imported by type


with
import List, { ClickEventDetail } from "./List.js";

ClickEventDetail is also imported as type, but if some changes ClickEventDetail to something else, it will be imported as value and we won't know.